src/mscorlib/src/System/Collections/Comparer.cs

```diff
@@ -41,18 +41,6 @@ public sealed class Comparer : IComparer , ISerializable
             Contract.EndContractBlock();
             m_compareInfo = culture.CompareInfo;
         }
-        
-        private Comparer(SerializationInfo info, StreamingContext context) {            
```

Comment:
It is used as internal implementation detail that cannot leak and thus cannot be serialized. It was renamed to LowLevelComparer (with more cruft stripped) in CoreRT. We should get it eventually as we are unifying the CoreCLR and CoreRT impls.